pkg/model/stack_serializer.go

```diff
@@ -1208,8 +1208,12 @@ func hasUnderscore(name string) bool {
 	return strings.Contains(name, "_")
 }
 
+func hasTrailingSpaces(name string) bool {
+	return strings.HasPrefix(name, " ") || strings.HasSuffix(name, " ")
+}
+
 func shouldBeSanitized(name string) bool {
-	return hasEmptySpace(name) || hasUnderscore(name) || hasUppercase(name)
+	return hasEmptySpace(name) || hasUnderscore(name) || hasUppercase(name) || hasTrailingSpaces(name)
```

Comment:
Nice catch

Reply:
If I understood correctly, we wouldn't need to check `hasTrailingSpaces` if we are already checking `hasEmptySpace`. Can you confirm @teresaromero ?

Reply:
that is correct! :) will upload the rever change